opencensus/proto/agent/common/v1/common.proto

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// NOTE: This proto is experimental and is subject to change at this point.
+// Please do not use it at the moment.
+
+package opencensus.proto.agent.common.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.proto.agent.common.v1";
+option java_outer_classname = "CommonProto";
+
+option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/agentproto/commonproto/v1";
+
+message Node {
+  // Identifier that uniquely identifies a process within a VM/container.
+  ProcessIdentifier identifier = 1;
+  
+  // Information on the OpenCensus Library who initiates the stream.
+  LibraryInfo LibraryInfo = 2;
+
+  // Additional attributes.
+  map<string, string> attributes = 3;
+  
+  // TODO(songya): Add more identifiers in the future as needed, like cloud/service identifiers.
+}
+  
+// Identifier that uniquely identifies a process within a VM/container.
+// In the future we plan to extend the identifier proto definition to support
+// additional information (e.g cloud id, monitored resource, service name, etc.)
+message ProcessIdentifier {
+  
+  // The host name.
+  string host_name = 1;
```

Comment:
what is a good example of host_name: machine/container name/ip address? Service name?

Reply:
Currently we use machine/container name, added some example.

Reply:
Thanks for the explanation. I was thinking about it more and it seems service name is kind of essential: e.g. zipkin and jaeger exporters explicitly ask for it, stackdriver exporters get it from env var and AppInsights also heavily rely on it. Since everyone needs them, can we have service name right away?

Reply:
That makes sense to me, we can add another `service_name` field to `Node`. WDYT @odeke-em @bogdandrutu

Reply:
I think we should have a "ServiceInfo" message which currently contains just "name", this way we can extend more the message if needed. I can think of version, build informations and others.

Reply:
SG, added a ServiceInfo message.